ArduCopter/AP_Arming.cpp

```diff
@@ -239,69 +239,21 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
             return false;
         }
 
-        // Check for 0 value PID's - some items can / should be 0 and as such are not checked.
-        // If the ATC_RAT_*_FF is non zero then the corresponding ATC_RAT_* PIDS can be 0.
-        if (is_zero(copter.pos_control->get_pos_xy_p().kP())) {
-            parameter_checks_pid_warning_message(display_failure, "PSC_POSXY_P");
+        // ensure controllers are OK with us arming:
+        char failure_msg[50];
+        if (!copter.pos_control->pre_arm_checks("PSC", failure_msg, ARRAY_SIZE(failure_msg))) {
+            check_failed(ARMING_CHECK_PARAMETERS, display_failure, "PosControl bad: %s", failure_msg);
```

Comment:
Should the prefix (here and AttControl) just be `Bad parameter:`? PosControl/AttControl won't mean much to users and this is a check for parameters, not a generic one.

Reply:
Changed.